src/how-tos/mqtt/mqtt.kafka.broker.md

```diff
@@ -217,24 +217,21 @@ With your `zilla.yaml` config, follow the [Zilla install instructions](../deploy
 @tab Docker
 
 ```bash:no-line-numbers
---env KAFKA_HOST="host.docker.internal" --env KAFKA_PORT="9092"
+--env KAFKA_BOOTSTRAP_SERVER="host.docker.internal:9092"
 ```
 
 @tab Helm values.yaml
 
 ```yaml:no-line-numbers
-extraEnv:
-  - name: KAFKA_HOST
-    value: "kafka.zilla-kafka-broker.svc.cluster.local"
-  - name: KAFKA_PORT
-    value: "9092"
+env:
+  KAFKA_BOOTSTRAP_SERVER: "kafka.zilla-kafka-broker.svc.cluster.local:9092"
 ```
 
 :::
 
 ### Adding TLS
 
-You can add TLS to this broker by adding a vault and tls binding as described in the [Server Encryption](../../concepts/config-intro.md#server-encryption-tls-ssl) section.
+You can add TLS to this broker by adding a vault and tls binding as described in the [Server Encryption](../../concepts/bindings.md#server-encryption-tls-ssl) section.
```

Comment:
```suggestion
You can add TLS to this MQTT broker by adding a vault and tls binding as described in the [Server Encryption](../../concepts/bindings.md#server-encryption-tls-ssl) section.
```

To disambiguate versus Kafka broker, since client connectivity to Kafka is right above and at first reading it seemed like a mistake to reference `Server Encryption` section instead of `Client Encryption` section, but now the intent is clear.